RecoLocalCalo/HGCalRecProducers/plugins/HGCalUncalibRecHitProducer.cc

```diff
@@ -70,11 +73,27 @@ HGCalUncalibRecHitProducer::produce(edm::Event& evt, const edm::EventSetup& es)
     worker_->run3(evt, itdg, *hebUncalibRechits);
   }
   
+  // loop over HFNose digis
+  edm::Handle< HGCalDigiCollection > pHGCHFNoseDigis;
+  evt.getByToken( hfnoseDigiCollection_, pHGCHFNoseDigis);
+  if (pHGCHFNoseDigis.isValid()) {
+    const HGCalDigiCollection* hfnoseDigis = 
+      pHGCHFNoseDigis.product(); // get a ptr to the product
```

Comment:
Either remove the comment, or put is aside the line of code, so that the line won't get automatically split by the clang-formatting